### ISSUE TITLE
Invalid Lore markup in `core/howto/clients`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4955]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4955|
|Type|enhancement|
|Created|2011-03-15 05:54:01Z|
|Branch|https://github.com/twisted/twisted/tree/invalid-markup-4955|

In the `core/howto/clients` document, there is an invalid tag:

```
<span class="note">
```

`class="note"` is only valid for <div> tags

<details><summary>Searchable metadata</summary>

```
trac-id__4955 4955
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__ 
branch__branches_invalid_markup_4955 branches/invalid-markup-4955
branch_author__khorn khorn
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1300168441000000 1300168441000000
changetime__1300170281000000 1300170281000000
version__None None
owner__khorn khorn

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_1"></a>|@khorn commented|
|-|-|

(In [31112]) Branching to 'invalid-markup-4955'

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn commented|
|-|-|

submitting for review

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set owner to @khorn|
|-|-|

The Lore documentation justifies the ticket summary, and the behavior is actually superior if you use a div instead of a span because of the css, despite the implementation of the output generator not really caring what kind of tag it is (`twisted.lore.tree.notes`).

+1, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_4"></a>|@khorn set status to `closed`|
|-|-|

(In [31118]) Merge invalid-markup-4955

Author: khorn
Reviewers: exarkun
Fixes: [#4955](https://github.com/twisted/twisted/issues/4955)

Fix "incorrect" markup in doc/core/howto/clients
